overrides/404.html

```diff
@@ -0,0 +1,49 @@
+{% extends "main.html" %}
+
+{% block content %}
+<style>
+    .md-sidebar--secondary:not([hidden]) {
+        display: none;
+    }
+
+    .md-content__button {
+        display: none;
+    }
+
+    h1 {
+        display: none;
+    }
+
+    .center-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        text-align: center;
+    }
+
+    .md-button {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: 20px;
+    }
+</style>
+
+<div class="center-container">
+    <h2>
+        Oops, looks like this page is still under construction
```

Comment:
also add that it might not exist

Reply:
Hello, thanks for the feedback on the 404 page. I'll go ahead and make those updates - adding text that the page might not exist. I'll push the changes shortly. Let me know if you need anything else on this!

Reply:
okie :3